games-puzzle/kpat/patches/kpat-17.12.1.patchset

```diff
@@ -0,0 +1,64 @@
+From 53df3fa42ad54fc0ee8933b89aa597e33f1f0b3d Mon Sep 17 00:00:00 2001
+From: Khaled Berraoui <khallebal@gmail.com>
+Date: Tue, 2 Jan 2018 21:32:27 +0100
+Subject: Disable Doctools and Dbus to fix start up crash
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8f738bf..588aff3 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -14,8 +14,8 @@ find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS
+     ConfigWidgets
+     CoreAddons
+     Crash
+-    DBusAddons
+-    DocTools
++#	DBusAddons
++#	DocTools
+     GuiAddons
+     I18n
+     KIO
+@@ -43,7 +43,7 @@ add_subdirectory(mimetypes)
+ add_subdirectory(previews)
+ add_subdirectory(sounds)
+ add_subdirectory(themes)
+-add_subdirectory(doc)
++#add_subdirectory(doc)
+ 
+ set(kpat_SRCS
+     main.cpp
+@@ -96,7 +96,7 @@ ecm_add_app_icon(kpat_SRCS ICONS ${ICONS_SRCS})
+ add_executable(kpat ${kpat_SRCS})
+ target_link_libraries(kpat
+     KF5::Crash
+-    KF5::DBusAddons
++#	KF5::DBusAddons
+     KF5::I18n
+     KF5::KIOCore
+     KF5KDEGames
+diff --git a/main.cpp b/main.cpp
+index c85c497..af1a6a8 100644
+--- a/main.cpp
++++ b/main.cpp
+@@ -48,7 +48,7 @@
+ 
+ #include <QDebug>
+ #include <KLocalizedString>
+-#include <KDBusService>
++//#include <KDBusService>
+ 
+ #include <QFile>
+ #include <QTime>
+@@ -332,7 +332,7 @@ int main( int argc, char **argv )
+     }
+     w->show();
+ 
+-    const KDBusService dbusService(KDBusService::Multiple);
++//    const KDBusService dbusService(KDBusService::Multiple);
```

Comment:
```diff
-+//    const KDBusService dbusService(KDBusService::Multiple);

Reply:
There is no need to remove them completely, we will just drop the patchset once the libs are fixed, what do you think?

Reply:
Sure, but I prefer small diff, the changes will be more obvious. But feel free to keep it as you wish, this is more or less only a suggestion